src/main/MQ2ImGuiConsole.cpp

```diff
@@ -1591,6 +1737,15 @@ static void ConsoleSettings()
 	ImGui::SameLine();
 	mq::imgui::HelpMarker("This feature allows you to automatically show the MacroQuest Console upon load.");
 
+	ImGui::SameLine();
+	if (ImGui::Checkbox("Developer: Command Logging", &s_consoleFileLog))
```

Comment:
Command Logging -> Command History

Reply:
History is used internally already -- CommandHistoryLog?